### DISCUSSION TITLE
New Devices

### DISCUSSION THREAD

**Discussion by kash1982** [tsightler/ring-mqtt#793] [Support Q&A]
The add on works perfectly and i can see all my devices and entities but ive added a new camera on the ring up yesterday but it hasnt been discovered yet. is there a way i can manually discover it?

**Comment by tsightler** [comment]
Device discovery is performed only during addon startup so you must restart the addon to discover and new devices, that's all.

**Comment by kash1982** [comment]
Thank you that worked perfectly.

**Comment by kmanan** [comment]
Just restarting? Or do I need to re-authenticate too?

**Reply by tsightler** [reply]
Just restart the addon (not Home Assistant), that is all.
